packages/amplication-client/src/Layout/PageContent.scss

```diff
@@ -1,5 +1,7 @@
 @import "../style/index.scss";
 
+$tabs-width: 300px;
```

Comment:
@barshimi @EugeneTseitlin how are we currently handling responsive design? 
Are we declaring measurements in `px`?

Reply:
it depends... if it's locally to the specific component/page we declare it as local variable, if it's not we added it to the global variables

Reply:
@overbit Depends on the situation. Not every component is responsive. This one, for example, is not. Left and right bars have static width, while middle part is responsive. We also have a min screen width, which is 900px, if I'm not mistaken. 
Regarding this specific change, that's how it was before, this morning I accepted PR that removed this line, so now I return it back.